features/enable_fips_vm.feature

```diff
@@ -487,47 +486,68 @@ Feature: FIPS enablement in lxd VMs
             fips +yes                enabled
             """
         When I reboot the `<release>` machine
-        And  I run `ua enable fips-updates --assume-yes` with sudo
+        And  I run `uname -r` as non-root
         Then stdout matches regexp:
             """
-            Updating package lists
-            Installing FIPS Updates packages
-            FIPS Updates enabled
-            A reboot is required to complete install
+            fips
             """
-        When I run `ua status --all` with sudo
+        When I verify that running `ua enable fips-updates --assume-yes` `with sudo` and stdin `y` exits `0`
```

Comment:
It is a bit confusing that we still need to answer an interactive prompt even with `--assume-yes`.

Also I think I'd rather see what happens without `--assume-yes` for the integration test IMO

Reply:
I have found some bugs related to the `assume-yes` flag. Thanks for spotting that.
However, since we have two prompts (One for fips-updates and one for disable), I couldn't use our default stdin to handle that, so I am keeping the `assume-yes` flag here

Reply:
I have managed to not use `--assume-yes` on the test. I think we should be ok now